sc-kpm/sc-agents-common/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+include_directories(${SC_KPM_SRC})
+
+set(SC_AGENTS_COMMON_SRC "${SC_KPM_ROOT}/sc-agents-common")
```

Comment:
This is not necessary. Better to not use absolute paths. Proposal is to remove `SC_AGENTS_COMMON_SRC`